Remove the 'stylex.attrs' API (#993)
facebook/stylex: packages/docs/docs/api/javascript/props.mdx

  )}
/>;
```

### Not using React?

:::tip For Solid, Svelte, Qwik, Vue

> For frameworks that expect `class` instead of `className`, and expect `style` to be a string instead of an object, you can
> wrap the `props` API and perform the relevant transform. Then use this function in place of `stylex.props`.

Reviewer: ```suggestion
For frameworks that expect `class` instead of `className`, and require `style` to be a string instead of an object,
wrap the `props` API with the function below and use it in place of `stylex.props`.
```
